src/setup/rgw.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python3
+#! /usr/bin/python3
```

Comment:
Why is this change ? /bin/env is more standard way to access python as it supports virtual env.

Reply:
rpm building was failing while compiling these python files.

Reply:
rpm building was failing while compiling these python files.